docs/integration_fetching_data.md

```diff
@@ -149,6 +149,10 @@ from datetime import timedelta
 SCAN_INTERVAL = timedelta(seconds=5)
 ```
 
+:::info
```

Comment:
I think we should put this info under a separate header so we can easily link to it. The current header where this note is placed is also not correct since this is not only for separate polling for each individual entity.

Reply:
Accepted but then the last paragraph with the code block is also not for individual entities.
To structure it I would introduce a section between "Coordinated ..." and "Separate ..." maybe "Setting a polling interval"? And move the last paragraph of "Separate .." and the code block in there along with the note?

Reply:
The last paragraph is correct. It's only for individually polled entities.